feat: map 'not enough coins' error (#2902)
FuelLabs/fuels-ts: apps/docs/src/guide/wallets/wallet-transferring.md

This section demonstrates additional examples of transferring assets between wallets and to contracts.

## Checking Balances

> Before transferring assets, ensure your wallet has sufficient funds. Attempting a transfer without enough funds will result in an error: `The transaction does not have enough funds to cover its execution.`.

Reviewer: Nit.

```suggestion
Before you transfer assets, please make sure your wallet has enough funds. Attempting a transfer without enough funds will result in the error: `The transaction does not have enough funds to cover its execution.`
```
